libs/wingsdk/src/target-tf-gcp/permissions.ts

```diff
@@ -42,3 +42,21 @@ export function calculateBucketPermissions(ops: string[]): string[] {
 
   return permissions;
 }
+
+export function calculateCounterPermissions(ops: string[]): string[] {
+  const permissions: string[] = [];
+
+  if (ops.includes(cloud.CounterInflightMethods.INC)) {
+    permissions.push("datastore.entities.get");
+  }
+
+  if (ops.includes(cloud.CounterInflightMethods.INC)) {
+    permissions.push("datastore.entities.create");
+  }
+
+  if (ops.includes(cloud.CounterInflightMethods.INC)) {
+    permissions.push("datastore.entities.update");
+  }
```

Comment:
why not put all three under the same condition? or at least split between get and create & update?

Reply:
It's still WIP, need to add the other methods.